src/EventEdition/GuestGrid.vala

```diff
@@ -106,8 +108,16 @@ public class Maya.View.EventEdition.GuestGrid : Gtk.Grid {
                         individual = map_iterator.get_value ();
                         if (individual != null) {
                             try {
-                                individual.avatar.load (32, null);
-                                avatar = new Granite.Widgets.Avatar.from_file (individual.avatar.to_string (), 32);
+                                individual.avatar.load (ICON_SIZE, null);
+                                avatar = new Hdy.Avatar (ICON_SIZE, individual.avatar.to_string (), true);
```

Comment:
Yes you might want to replace these lines with something like:
```suggestion
                                avatar = new Hdy.Avatar (ICON_SIZE, individual.display_name, true);
```

Reply:
I've checked Folks.Individual doc again and I thinks that's correct. Thanks!